util/nvticache.c

```diff
@@ -363,12 +231,11 @@ nvticache_add (const nvti_t *nvti, const char *filename)
 char *
 nvticache_get_src (const char *oid)
 {
-  char *filename, *src, pattern[2048];
+  char *filename, *src;
 
   assert (cache_kb);
 
-  g_snprintf (pattern, sizeof (pattern), "oid:%s:filename", oid);
-  filename = kb_item_get_str (cache_kb, pattern);
+  filename = kb_nvt_get (cache_kb, oid, 0);
```

Comment:
I think using a `#DEFINE` or other enumeration of indices would improve readability and maintainability.